lambdas/checkpassport/src/main/java/uk/gov/di/ipv/cri/passport/checkpassport/handler/CheckPassportHandler.java

```diff
@@ -130,6 +130,11 @@ private void initializeLambdaServices(
 
         this.thirdPartyAPIServiceFactory = new ThirdPartyAPIServiceFactory(serviceFactory);
 
+        // Prime DynamoDB Client
+        SessionItem primedSessionItem = sessionService.getSession("-1");
```

Comment:
Can you explain how the -1 works here just for my own understanding?

Reply:
We never want to actually get a real session id, just trigger the code path way so any classes and objects created get snapshotted as well.